src/hotspot/cpu/x86/c2_MacroAssembler_x86.hpp

```diff
@@ -31,8 +31,8 @@
   Assembler::AvxVectorLen vector_length_encoding(int vlen_in_bytes);
 
   // special instructions for EVEX
-  void setvectmask(Register dst, Register src);
-  void restorevectmask();
+  void setvectmask(Register dst, Register src, KRegister mask = k1);
+  void restorevectmask(KRegister mask = k1);
```

Comment:
I don't see much sense in default values here. The register should be explicitly specified at call sites.